engine/actor.py

```diff
@@ -5,134 +5,135 @@
 from engine import item
 from engine import say
 from engine import tartarus
-
 from cronenbroguelike import util
 
 
 class _Statistic:
```

Comment:
I feel like `current_value` should be a property in this class that the subclasses (`_VariableStatistic` and `_StaticStatistic`) override if needed. It could also simply be called `value`.